packages/editor/src/components/page-attributes/README.md

```diff
@@ -0,0 +1,160 @@
+Here components that provide the functionality available in `Page Attributes` panel of the document sidebar are documented.
+Using these components, one can add a UI that allows changing page attributes in different scenarios/needs.
+
+PageAttributesCheck
+===================
+
+This component is used to check if the page attributes section should be rendered or not.
```

Comment:
I think generally if these lines are intended to be separate paragraphs there should be a newline between each line. Otherwise, they can all be one line (we don't do line-wrapping in markdown).